README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Overview
```

Comment:
Need to update this as well https://github.com/primevprotocol/mev-commit/blob/main/pkg/abi/abi.go#L1244 and run the integration test before merging this in